### PR TITLE
Add "Teamwork Desk" extension

### PR DESCRIPTION
**Pull request by barisdemirhan**
## Description

List all Teamwork Desk tickets and quickly access from Raycast.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/17131004/152887951-4051d23c-e0b7-4946-9a77-e79f72c983e4.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/teamwork-desk/package.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  "title": "Teamwork Desk",
+  "description": "List all tickets and quickly access from Raycast.",
+  "icon": "teamwork-desk-icon.png",
+  "author": "barisdemirhan",
+  "version": "1.0.1",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Teamwork Desk",
```
Comment (pernielsentikaer) [inline extensions/teamwork-desk/package.json:13]:
```suggestion
      "title": "List Tickets",
       "subtitle": "Teamwork Desk",
```

**Review by pernielsentikaer — COMMENTED** [review]
Hi @barisdemirhan 👋 

Thanks for your contribution, it looks very nice 💪 

I'm having problems getting it to work, could you find me en [Slack](https://raycastcommunity.slack.com/) or another preferred channel to talk about it?

One thing I know is that I opened an account in EU, so my URL looks like this:
https://xxx.eu.teamwork.com/ I just added xxx and entered API key – I'm getting 404 response back

It would also make sense to add a README telling the user how to get the necessary data in order to use the extension.

Looking forward to hearing from you again 💯

**Comment by pernielsentikaer** [comment]
> > Hi @barisdemirhan 👋
> > Thanks for your contribution, it looks very nice 💪
> > I'm having problems getting it to work, could you find me en [Slack](https://raycastcommunity.slack.com/) or another preferred channel to talk about it?
> > One thing I know is that I opened an account in EU, so my URL looks like this: https://xxx.eu.teamwork.com/ I just added xxx and entered API key – I'm getting 404 response back
> > It would also make sense to add a README telling the user how to get the necessary data in order to use the extension.
> > Looking forward to hearing from you again 💯
> 
> Hi @pernielsentikaer, thank you for your feedback. I didn't know that the link changed like this when the EU account was opened. Probably adding the domain with .eu will fix the problem, I'll check it out. You're also right, it would be useful to create a README.
> 
> I will make an update soon.

You can get my test account if that's help you 😄

**Comment by fe9lix** [comment]
Hey @barisdemirhan, what the status of your PR? Btw, the CI is failing with the package-lock.json file (looks like you should upgrade to npm v7...)

**Comment by sxn** [comment]
hi @barisdemirhan - I'll close this PR due to inactivity. Feel free to reach out to any of us once you've had a bit of time to work on the changes, or open a new PR. 😄

**Pull request closed without merging**